src/Components/DynamicContent/Search/Search.js

```diff
@@ -0,0 +1,92 @@
+
+import React, { Component } from 'react';
+
+import SearchResult from './SearchResult'
+
+
+class Search extends Component {
+
+    state = {
+        searchTerm:"",
+        searchDisplay:[]
+    };
+
+    promotionData = {
+        "promotions" : [
+            {"name":"lapse Prevention", "startDate": "01 Jan 2019", "endDate": "20 Jan 2019"},
+            {"name":"Welcome", "startDate": "03 Jan 2019", "endDate": "20 Feb 2019"},
+            {"name":"Welcome Back", "startDate": "06 Jan 2019", "endDate": "22 Jan 2019"},
+            {"name":"Cash Back", "startDate": "18 Jan 2019", "endDate": "30 Jan 2019"},
+            {"name":"Play Day", "startDate": "01 Jan 2019", "endDate": "01 Mar 2019"},
+            {"name":"New Play Day", "startDate": "01 Jan 2019", "endDate": "01 Mar 2019"}
+        ]
+    }
+
+    handleInputChange = (event) => {
```

Comment:
👍 good function name, no need for brackets around the event as it is a single parameter